src/Core/Models/View/SendTextView.cs

```diff
@@ -0,0 +1,17 @@
+using Bit.Core.Models.Domain;
+
+namespace Bit.Core.Models.View
+{
+    public class SendTextView : View
+    {
+        public SendTextView() { }
+        public SendTextView(SendText text)
+        {
+            Hidden = text.Hidden;
+        }
+
+        public string Text { get; set; } = null;
+        public bool Hidden { get; set; }
+        public string MaskedText => Text != null ? "••••••••" : null;
```

Comment:
Maybe change `Text != null` check to `!string.IsNullOrEmpty(Text)` (whitespace is the norm, however here white-space could be a password I guess (funny password, all spaces, but you never know I guess)

Reply:
I'm hesitant include empty for the same reason. I copied the behavior from [`LoginView`'s MaskedPassword member](https://github.com/bitwarden/mobile/blob/master/src/Core/Models/View/LoginView.cs#L23).

Reply:
That's fair, let's leave it for now. Thanks!